extensions/wikia/GlobalNavigation/scripts/GlobalNavigationExploreWikiaDropdown.js

```diff
@@ -0,0 +1,49 @@
+require(
+	[
+		'jquery', 'wikia.window', 'wikia.delayedhover', 'wikia.globalNavigationDropdowns'
+	],
+	function ($, w, delayedHover, dropdowns) {
+		'use strict';
+
+		var $entryPoint,
+			$exploreLink;
+
+		/**
+		 * Handle click on entry point for logged in users.
+		 * Second click on entry point for logged in users is redirecting to user profile page.
+		 * This method should be removed after we unify the ux for anon and logged in.
+		 * @param {Event} event
+		 */
+		function onEntryPointClick(event) {
+			var $this = $(event.target);
+
+			event.preventDefault();
+			event.stopImmediatePropagation();
+
+			if ($entryPoint.hasClass('active')) {
```

Comment:
Does it work because on touch devices there is no active class on the first click? Would be nice to have a comment here explaining what's going on.